src/status/wallet.nim

```diff
@@ -187,3 +187,21 @@ proc validateMnemonic*(self: WalletModel, mnemonic: string): string =
   result = status_wallet.validateMnemonic(mnemonic).parseJSON()["error"].getStr
 
 proc getAllCollectibles*(self: WalletModel, address: string): seq[Collectible] = getAllCollectibles(address)
+
+proc getGasPricePredictions*(self: WalletModel): GasPricePrediction =
+  try:
+    let url: string = fmt"https://etherchain.org/api/gasPriceOracle"
```

Comment:
Went with etherchain.org oral for now, prolly wanna do something else in the future